src/Phinx/Db/Adapter/SqlServerAdapter.php

```diff
@@ -1204,12 +1204,13 @@ public function hasDatabase(string $name): bool
      */
     public function dropDatabase(string $name): void
     {
-        $sql = <<<SQL
-USE master;
-IF EXISTS(select * from sys.databases where name=N'$name')
-ALTER DATABASE [$name] SET SINGLE_USER WITH ROLLBACK IMMEDIATE;
-DROP DATABASE [$name];
-SQL;
+        $sql = sprintf(
+            'USE master;
+            IF EXISTS(select * from sys.databases where name=N\'' . $name . '\')
+            ALTER DATABASE %1$s SET SINGLE_USER WITH ROLLBACK IMMEDIATE;
+            DROP DATABASE %1$s;',
```

Comment:
By using `%1$s`, I'm able to replace both instances with a single value. Downside is that cannot use `"` around the string like do in other places. Could use a heredoc, and then run the `$sql` through `sprintf` on a separate line, but that didn't feel in line with other places `sprintf` is used.